javabuilder-authorizer/token_validator.rb

```diff
@@ -202,8 +221,19 @@ def user_over_limit?(time_range_seconds, limit, logging_message)
     if response.last_evaluated_key
       puts "user_requests query has paginated responses. user_id #{@user_id}"
     end
+    response
+  end
+
+  def user_near_limit?(count, limit)
```

Comment:
nit: this reads a little weird to me because it returns a hash in the true case (rather than a boolean). Would it make more sense for this to be something like `get_near_limit_detail` and return `nil` if it's not near the limit?

Reply:
+1

Reply:
done!